### PR TITLE
Add collect docs to the docsv2 project

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Five new documents: 

- How Netdata's metrics collectors work
- Enable or configure a collector
- Collect system metrics with Netdata
- Collect container metrics with Netdata
- Collect application metrics with Netdata

> See netdata/netdata-learn-docusaurus#285 for additional context.
>
> This page contains some elements that will only appear on the deployed Learn site, so see the deploy preview for this page for the full picture: https://deploy-preview-285--netdata-docusaurus.netlify.app/docs/get
>
> Some links will not work, as the target docs do not exist yet.
>
> For now, these PRs will get merged into the docsv2 but not go live on Netdata Learn. Once they're all available, or at least a workable majority, we'll merge them into master and deploy the project as a whole.

##### Component Name

area/docs

### CODE REVIEW

File: docs/collect/enable-configure.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!--
+title: "Enable or configure a collector"
+description: "Every collector is highly configurable, allowing them to collect metrics from any node and any infrastructure."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/collect/enable-configure.md
+-->
+
+# Enable or configure a collector
+
+While collectors start colleting right away if they find exposed metrics on their default endpoint, not all
```
Comment (aabatangle) [inline docs/collect/enable-configure.md:9]: typo - colleting-->collecting

File: docs/collect/how-collectors-work.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+-   [Containers](/docs/collect/container-metrics.md): Gather metrics from container agents, like `dockerd` or `kubectl`,
+    along with the resource usage of containers and the applications they run.
+-   [Applications](/docs/collect/application-metrics.md): Collect per-second metrics from web servers, databases, logs,
+    message brokers, APM tools, email servers, and much more.
+
+## Collector architecture and terminology
+
+**Collector** is a catch-all term for any Netdata process that gathers metrics from an endpoint. 
+
+While we use collector most often in documentation, release notes, and educational content, you may encounter other
```
Comment (aabatangle) [inline docs/collect/how-collectors-work.md:53]: Suggest: While we use 'collector' most often

File: docs/collect/system-metrics.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/collect/system-metrics.md
+-->
+
+# Collect system metrics with Netdata
+
+Netdata collects thousands of metrics directly the operating systems of physical and virtual systems, IoT/edge devices,
+and [containers](/docs/collect/container-metrics.md) with zero configuration.
+
+To gather system metrics, Netdata uses roughly a dozen plugins, each of which has one or more collectors for very
+specific metrics exposed by the host. Most metrics Netdata users interface with most for health monitoring and
```
Comment (aabatangle) [inline docs/collect/system-metrics.md:14]: Suggest changing "users interface" to "users interact". I stumbled on this, reading it as "user interface".

**Review by aabatangle — APPROVED** [review]

**Review by aabatangle — APPROVED** [review]

File: docs/collect/system-metrics.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<!--
+title: "Collect system metrics with Netdata"
+sidebar_label: "System metrics"
+description: "Netdata collects thousands of metrics from physical and virtual systems, IoT/edge devices, and containers with zero configuration."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/collect/system-metrics.md
+-->
+
+# Collect system metrics with Netdata
+
+Netdata collects thousands of metrics directly the operating systems of physical and virtual systems, IoT/edge devices,
```
Comment (jlbriston) [inline docs/collect/system-metrics.md:10]:
```suggestion
Netdata collects thousands of metrics directly from the operating systems of physical and virtual systems, IoT/edge devices,
```

**Review by jlbriston — APPROVED** [review]

**Pull request merged**